wof/wofpy_config.py

```diff
@@ -14,6 +16,7 @@
 
 Usage:
     wofpy_config INDIR
+    wofpy_config INDIR [--mode=<test,production>]
```

Comment:
@lsetiawan is this OK for you? Like you requested `--mode=test` is the default and can be omitted.
(Although it is odd to have a "test" as default, but I am not good with names so I don't have anything better to suggest :grimacing:)